core/src/main/scalajs/sttp/tapir/TapirFile.scala

```diff
@@ -0,0 +1,14 @@
+package sttp.tapir
+
+import java.nio.file.Path
+
+case class TapirFile(underlying: Any, range: Option[RangeValue] = None) {
+  def toPath: Path = underlying.asInstanceOf[Path]
+  def toFile: java.io.File = toPath.toFile
```

Comment:
similarly, this can't be present in the shared sources. We need a `TapirFileExtensions` which will have platform-specific methods